model/task/db.go

```diff
@@ -20,6 +20,15 @@ const (
 	OldCollection = "old_tasks"
 )
 
+var (
+	Distro_1_status_1_activated_1_priority_1 = bson.D{
```

Comment:
Can we also call this something more userfriendly?  In this case ActivatedTasksByDistroIndex or UnderwaterTasksIndex or etc

Reply:
Really "Index" is important here; as it is this isn't any more readable than hardcoding it.